Logos/AzurePurview.svg

```diff
@@ -0,0 +1,33 @@
+<path d="M6.158 10.5C5.82588 9.88832 5.70383 9.18449 5.8106 8.4967C5.91737 7.8089 6.24706 7.1752 6.749 6.693C7.55962 5.60185 8.69594 4.79644 9.994 4.393C10.7017 4.1769 11.4362 4.06111 12.176 4.049C11.5808 3.79106 10.9576 3.60313 10.319 3.489C8.57029 3.19461 6.77333 3.49123 5.212 4.332C4.33842 4.7851 3.51404 5.32732 2.752 5.95C3.1117 7.32294 3.83754 8.57241 4.852 9.565C5.24328 9.93401 5.68258 10.2485 6.158 10.5V10.5Z" fill="url(#fd386209-a692-4bef-b7d5-514d43f4e5fa)"/>
+<path d="M17.73 8.379C16.7528 7.20844 15.6242 6.17303 14.374 5.3C13.0667 4.94302 11.6867 4.94993 10.383 5.32C9.99125 5.43923 9.61342 5.60015 9.256 5.8C9.81451 5.83934 10.353 6.02413 10.818 6.336C10.8597 6.34793 10.8994 6.36578 10.936 6.389C11.7055 6.90574 12.3597 7.57626 12.8572 8.35832C13.3548 9.14038 13.685 10.017 13.827 10.933C13.9186 11.6512 13.9075 12.3789 13.794 13.094C15.2758 12.1523 16.6023 10.986 17.726 9.637C17.8123 9.55665 17.8812 9.45945 17.9284 9.3514C17.9757 9.24336 18.0002 9.12678 18.0006 9.00887C18.001 8.89095 17.9771 8.77422 17.9306 8.66588C17.8841 8.55754 17.8158 8.4599 17.73 8.379V8.379Z" fill="url(#poh86209-a692-4bef-b7d5-514d43f4e5fa)"/>
+<path d="M21.379 -3.738H-2.9V24.033H21.379V-3.738Z" stroke="#B31B1B" stroke-miterlimit="10"/>
+</g>
+<defs>
+<linearGradient id="d72d6332-1ed4-4600-9e81-04168c50066c" x1="8.977" y1="11.134" x2="8.977" y2="7.178" gradientUnits="userSpaceOnUse">
+<stop stop-color="#29BADE"/>
+<stop offset="1" stop-color="#50E6FF"/>
+</linearGradient>
+<linearGradient id="11a05856-62b4-4aad-b49b-76e766082268" x1="6.446" y1="14.626" x2="6.446" y2="3.374" gradientUnits="userSpaceOnUse">
+<stop offset="0.041" stop-color="#32BEDD"/>
+<stop offset="0.495" stop-color="#50E6FF"/>
+<stop offset="0.92" stop-color="#C3F1FF"/>
+</linearGradient>
+<linearGradient id="fd386209-a692-4bef-b7d5-514d43f4e5fa" x1="7.464" y1="10.047" x2="7.464" y2="0.254" gradientUnits="userSpaceOnUse">
+<stop offset="0.244" stop-color="#0078D4"/>
+<stop offset="0.6" stop-color="#2E8CE1"/>
+<stop offset="0.933" stop-color="#589EED"/>
+</linearGradient>
+<linearGradient id="poh86209-a692-4bef-b7d5-514d43f4e5fa" x1="13.628" y1="14.626" x2="13.628" y2="4.193" gradientUnits="userSpaceOnUse">
```

Comment:
Tests are failing because this is not a valid GUID. Please change this and its references.